[
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.VqIB7UrHis"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:25 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:27 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:28 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:36 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:38 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:39 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:47 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:49 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:50 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "mkdir(): File exists in /wordpress/wp-content/plugins/products-csv-importer-for-woocommerce/products-csv-importer.php on line 77",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:49:03 UTC] PHP Warning:  mkdir(): File exists in /wordpress/wp-content/plugins/products-csv-importer-for-woocommerce/products-csv-importer.php on line 77",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "mkdir(): File exists in /wordpress/wp-content/plugins/products-csv-importer-for-woocommerce/products-csv-importer.php on line 77",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:49:12 UTC] PHP Warning:  mkdir(): File exists in /wordpress/wp-content/plugins/products-csv-importer-for-woocommerce/products-csv-importer.php on line 77",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:33 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:34 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:42 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT pf_synctime FROM wp_petfinder_404 Limit 1</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT pf_synctime FROM wp_petfinder_404 Limit 1 | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3031): WP_SQLite_DB->query('SELECT pf_synct...')\n#2 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(163): wpdb->get_var('SELECT pf_synct...')\n#3 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(22): petfinder404->lookup_lastsync()\n#4 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(182): petfinder404->__construct()\n#5 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('pf404-for-petfi...')\n#7 /internal/eval.php(20): activate_plugin('pf404-for-petfi...')\n#8 {main}</pre>\n for query SELECT pf_synctime FROM wp_petfinder_404 Limit 1 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/pf404-for-petfinder/plugin.php'), petfinder404->__construct, petfinder404->lookup_lastsync, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:42 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>Delete From wp_petfinder_404</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: Delete From wp_petfinder_404 | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(146): WP_SQLite_DB->query('Delete From wp_...')\n#2 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(92): petfinder404::delete_table()\n#3 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(176): petfinder404::load_api_data()\n#4 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(22): petfinder404->lookup_lastsync()\n#5 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(182): petfinder404->__construct()\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('pf404-for-petfi...')\n#8 /internal/eval.php(20): activate_plugin('pf404-for-petfi...')\n#9 {main}</pre>\n for query Delete From wp_petfinder_404 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/pf404-for-petfinder/plugin.php'), petfinder404->__construct, petfinder404->lookup_lastsync, petfinder404::load_api_data, petfinder404::delete_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:42 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:43 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:51 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT pf_synctime FROM wp_petfinder_404 Limit 1</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT pf_synctime FROM wp_petfinder_404 Limit 1 | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3031): WP_SQLite_DB->query('SELECT pf_synct...')\n#2 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(163): wpdb->get_var('SELECT pf_synct...')\n#3 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(22): petfinder404->lookup_lastsync()\n#4 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(182): petfinder404->__construct()\n#5 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('pf404-for-petfi...')\n#7 /internal/eval.php(20): activate_plugin('pf404-for-petfi...')\n#8 {main}</pre>\n for query SELECT pf_synctime FROM wp_petfinder_404 Limit 1 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/pf404-for-petfinder/plugin.php'), petfinder404->__construct, petfinder404->lookup_lastsync, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:51 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>Delete From wp_petfinder_404</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: Delete From wp_petfinder_404 | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(146): WP_SQLite_DB->query('Delete From wp_...')\n#2 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(92): petfinder404::delete_table()\n#3 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(176): petfinder404::load_api_data()\n#4 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(22): petfinder404->lookup_lastsync()\n#5 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(182): petfinder404->__construct()\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('pf404-for-petfi...')\n#8 /internal/eval.php(20): activate_plugin('pf404-for-petfi...')\n#9 {main}</pre>\n for query Delete From wp_petfinder_404 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/pf404-for-petfinder/plugin.php'), petfinder404->__construct, petfinder404->lookup_lastsync, petfinder404::load_api_data, petfinder404::delete_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:51 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:52 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #5 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"photo-discussion\"},\"options\":{\"activate\":true}}) : Could not download \"https://downloads.wordpress.org/plugin/photo-discussion.latest-stable.zip\".\n  \t\t\t\tCheck if the URL is correct and the server is reachable.\n  \t\t\t\tIf it is reachable, the server might be blocking the request.\n  \t\t\t\tCheck the browser console and network tabs for more information.\n  \n  \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n  \n  \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n  \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n  \t\t\t\tcross-origin file downloads. Learn more about CORS at\n  \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n  \n  \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n  \t\t\t\tHere's how to do that:\n  \n  \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n  \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n  \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n  \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n  \n  \t\t\t\tThe resulting URL should look like this:\n  \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n  \n  \t\t\t\tError:\n  \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/photo-discussion.latest-stable.zip\"\n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: Error: Could not download \"https://downloads.wordpress.org/plugin/photo-discussion.latest-stable.zip\".\n    \t\t\t\tCheck if the URL is correct and the server is reachable.\n    \t\t\t\tIf it is reachable, the server might be blocking the request.\n    \t\t\t\tCheck the browser console and network tabs for more information.\n    \n    \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n    \n    \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n    \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n    \t\t\t\tcross-origin file downloads. Learn more about CORS at\n    \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n    \n    \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n    \t\t\t\tHere's how to do that:\n    \n    \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n    \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n    \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n    \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n    \n    \t\t\t\tThe resulting URL should look like this:\n    \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n    \n    \t\t\t\tError:\n    \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/photo-discussion.latest-stable.zip\"\n        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)\n  }\n}\n\n",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.3eQdf5YRwh"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:25 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:27 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:28 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:36 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:38 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:39 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:47 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:49 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:50 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "mkdir(): File exists in /wordpress/wp-content/plugins/products-csv-importer-for-woocommerce/products-csv-importer.php on line 77",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:49:03 UTC] PHP Warning:  mkdir(): File exists in /wordpress/wp-content/plugins/products-csv-importer-for-woocommerce/products-csv-importer.php on line 77",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "mkdir(): File exists in /wordpress/wp-content/plugins/products-csv-importer-for-woocommerce/products-csv-importer.php on line 77",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:49:12 UTC] PHP Warning:  mkdir(): File exists in /wordpress/wp-content/plugins/products-csv-importer-for-woocommerce/products-csv-importer.php on line 77",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:33 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:34 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:42 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT pf_synctime FROM wp_petfinder_404 Limit 1</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT pf_synctime FROM wp_petfinder_404 Limit 1 | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3031): WP_SQLite_DB->query('SELECT pf_synct...')\n#2 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(163): wpdb->get_var('SELECT pf_synct...')\n#3 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(22): petfinder404->lookup_lastsync()\n#4 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(182): petfinder404->__construct()\n#5 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('pf404-for-petfi...')\n#7 /internal/eval.php(20): activate_plugin('pf404-for-petfi...')\n#8 {main}</pre>\n for query SELECT pf_synctime FROM wp_petfinder_404 Limit 1 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/pf404-for-petfinder/plugin.php'), petfinder404->__construct, petfinder404->lookup_lastsync, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:42 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>Delete From wp_petfinder_404</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: Delete From wp_petfinder_404 | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(146): WP_SQLite_DB->query('Delete From wp_...')\n#2 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(92): petfinder404::delete_table()\n#3 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(176): petfinder404::load_api_data()\n#4 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(22): petfinder404->lookup_lastsync()\n#5 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(182): petfinder404->__construct()\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('pf404-for-petfi...')\n#8 /internal/eval.php(20): activate_plugin('pf404-for-petfi...')\n#9 {main}</pre>\n for query Delete From wp_petfinder_404 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/pf404-for-petfinder/plugin.php'), petfinder404->__construct, petfinder404->lookup_lastsync, petfinder404::load_api_data, petfinder404::delete_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:42 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:43 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:51 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT pf_synctime FROM wp_petfinder_404 Limit 1</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT pf_synctime FROM wp_petfinder_404 Limit 1 | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3031): WP_SQLite_DB->query('SELECT pf_synct...')\n#2 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(163): wpdb->get_var('SELECT pf_synct...')\n#3 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(22): petfinder404->lookup_lastsync()\n#4 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(182): petfinder404->__construct()\n#5 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('pf404-for-petfi...')\n#7 /internal/eval.php(20): activate_plugin('pf404-for-petfi...')\n#8 {main}</pre>\n for query SELECT pf_synctime FROM wp_petfinder_404 Limit 1 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/pf404-for-petfinder/plugin.php'), petfinder404->__construct, petfinder404->lookup_lastsync, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:51 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>Delete From wp_petfinder_404</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: Delete From wp_petfinder_404 | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(146): WP_SQLite_DB->query('Delete From wp_...')\n#2 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(92): petfinder404::delete_table()\n#3 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(176): petfinder404::load_api_data()\n#4 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(22): petfinder404->lookup_lastsync()\n#5 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(182): petfinder404->__construct()\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('pf404-for-petfi...')\n#8 /internal/eval.php(20): activate_plugin('pf404-for-petfi...')\n#9 {main}</pre>\n for query Delete From wp_petfinder_404 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/pf404-for-petfinder/plugin.php'), petfinder404->__construct, petfinder404->lookup_lastsync, petfinder404::load_api_data, petfinder404::delete_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:51 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:52 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #1 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"photo-discussion\"},\"options\":{\"activate\":true}}) : Could not download \"https://downloads.wordpress.org/plugin/photo-discussion.latest-stable.zip\".\n  \t\t\t\tCheck if the URL is correct and the server is reachable.\n  \t\t\t\tIf it is reachable, the server might be blocking the request.\n  \t\t\t\tCheck the browser console and network tabs for more information.\n  \n  \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n  \n  \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n  \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n  \t\t\t\tcross-origin file downloads. Learn more about CORS at\n  \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n  \n  \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n  \t\t\t\tHere's how to do that:\n  \n  \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n  \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n  \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n  \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n  \n  \t\t\t\tThe resulting URL should look like this:\n  \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n  \n  \t\t\t\tError:\n  \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/photo-discussion.latest-stable.zip\"\n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: Error: Could not download \"https://downloads.wordpress.org/plugin/photo-discussion.latest-stable.zip\".\n    \t\t\t\tCheck if the URL is correct and the server is reachable.\n    \t\t\t\tIf it is reachable, the server might be blocking the request.\n    \t\t\t\tCheck the browser console and network tabs for more information.\n    \n    \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n    \n    \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n    \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n    \t\t\t\tcross-origin file downloads. Learn more about CORS at\n    \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n    \n    \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n    \t\t\t\tHere's how to do that:\n    \n    \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n    \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n    \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n    \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n    \n    \t\t\t\tThe resulting URL should look like this:\n    \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n    \n    \t\t\t\tError:\n    \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/photo-discussion.latest-stable.zip\"\n        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)\n  }\n}\n\n",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.caHjzuhf4r"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:25 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:27 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:28 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:36 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:38 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:39 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:47 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:49 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:47:50 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>pro-mime-types</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "mkdir(): File exists in /wordpress/wp-content/plugins/products-csv-importer-for-woocommerce/products-csv-importer.php on line 77",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:49:03 UTC] PHP Warning:  mkdir(): File exists in /wordpress/wp-content/plugins/products-csv-importer-for-woocommerce/products-csv-importer.php on line 77",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "mkdir(): File exists in /wordpress/wp-content/plugins/products-csv-importer-for-woocommerce/products-csv-importer.php on line 77",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:49:12 UTC] PHP Warning:  mkdir(): File exists in /wordpress/wp-content/plugins/products-csv-importer-for-woocommerce/products-csv-importer.php on line 77",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:33 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:34 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:42 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT pf_synctime FROM wp_petfinder_404 Limit 1</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT pf_synctime FROM wp_petfinder_404 Limit 1 | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3031): WP_SQLite_DB->query('SELECT pf_synct...')\n#2 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(163): wpdb->get_var('SELECT pf_synct...')\n#3 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(22): petfinder404->lookup_lastsync()\n#4 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(182): petfinder404->__construct()\n#5 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('pf404-for-petfi...')\n#7 /internal/eval.php(20): activate_plugin('pf404-for-petfi...')\n#8 {main}</pre>\n for query SELECT pf_synctime FROM wp_petfinder_404 Limit 1 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/pf404-for-petfinder/plugin.php'), petfinder404->__construct, petfinder404->lookup_lastsync, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:42 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>Delete From wp_petfinder_404</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: Delete From wp_petfinder_404 | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(146): WP_SQLite_DB->query('Delete From wp_...')\n#2 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(92): petfinder404::delete_table()\n#3 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(176): petfinder404::load_api_data()\n#4 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(22): petfinder404->lookup_lastsync()\n#5 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(182): petfinder404->__construct()\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('pf404-for-petfi...')\n#8 /internal/eval.php(20): activate_plugin('pf404-for-petfi...')\n#9 {main}</pre>\n for query Delete From wp_petfinder_404 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/pf404-for-petfinder/plugin.php'), petfinder404->__construct, petfinder404->lookup_lastsync, petfinder404::load_api_data, petfinder404::delete_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:42 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:43 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:51 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SELECT pf_synctime FROM wp_petfinder_404 Limit 1</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: SELECT pf_synctime FROM wp_petfinder_404 Limit 1 | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3031): WP_SQLite_DB->query('SELECT pf_synct...')\n#2 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(163): wpdb->get_var('SELECT pf_synct...')\n#3 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(22): petfinder404->lookup_lastsync()\n#4 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(182): petfinder404->__construct()\n#5 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#6 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('pf404-for-petfi...')\n#7 /internal/eval.php(20): activate_plugin('pf404-for-petfi...')\n#8 {main}</pre>\n for query SELECT pf_synctime FROM wp_petfinder_404 Limit 1 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/pf404-for-petfinder/plugin.php'), petfinder404->__construct, petfinder404->lookup_lastsync, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:51 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>Delete From wp_petfinder_404</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: Delete From wp_petfinder_404 | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_petfinder_404.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(146): WP_SQLite_DB->query('Delete From wp_...')\n#2 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(92): petfinder404::delete_table()\n#3 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(176): petfinder404::load_api_data()\n#4 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(22): petfinder404->lookup_lastsync()\n#5 /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php(182): petfinder404->__construct()\n#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('pf404-for-petfi...')\n#8 /internal/eval.php(20): activate_plugin('pf404-for-petfi...')\n#9 {main}</pre>\n for query Delete From wp_petfinder_404 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/pf404-for-petfinder/plugin.php'), petfinder404->__construct, petfinder404->lookup_lastsync, petfinder404::load_api_data, petfinder404::delete_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:51 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "photo-discussion",
    "details": "[23-Apr-2025 23:50:52 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/pf404-for-petfinder/plugin.php on line 94\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #1 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"photo-discussion\"},\"options\":{\"activate\":true}}) : Could not download \"https://downloads.wordpress.org/plugin/photo-discussion.latest-stable.zip\".\n  \t\t\t\tCheck if the URL is correct and the server is reachable.\n  \t\t\t\tIf it is reachable, the server might be blocking the request.\n  \t\t\t\tCheck the browser console and network tabs for more information.\n  \n  \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n  \n  \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n  \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n  \t\t\t\tcross-origin file downloads. Learn more about CORS at\n  \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n  \n  \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n  \t\t\t\tHere's how to do that:\n  \n  \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n  \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n  \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n  \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n  \n  \t\t\t\tThe resulting URL should look like this:\n  \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n  \n  \t\t\t\tError:\n  \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/photo-discussion.latest-stable.zip\"\n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: Error: Could not download \"https://downloads.wordpress.org/plugin/photo-discussion.latest-stable.zip\".\n    \t\t\t\tCheck if the URL is correct and the server is reachable.\n    \t\t\t\tIf it is reachable, the server might be blocking the request.\n    \t\t\t\tCheck the browser console and network tabs for more information.\n    \n    \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n    \n    \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n    \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n    \t\t\t\tcross-origin file downloads. Learn more about CORS at\n    \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n    \n    \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n    \t\t\t\tHere's how to do that:\n    \n    \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n    \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n    \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n    \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n    \n    \t\t\t\tThe resulting URL should look like this:\n    \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n    \n    \t\t\t\tError:\n    \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/photo-discussion.latest-stable.zip\"\n        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)\n  }\n}\n\n",
    "log": "/logs/plugins/p/photo-discussion/error.json"
  }
]
